app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,9 @@
   </head>
 
   <body>
-  <h1>Little Esty Shop</h1>
+    <div style="text-align: center; color: blue;">
```

Comment:
Thanks for doing this!
I tried something similar but it wasn't working. Because I didn't but the styling in a div tag. I was trying it inside the header  tag. *facepalm*